app/components/HouseholderForm/HouseholderIdentityForm.tsx

```diff
@@ -117,13 +113,20 @@ export const HouseholderIdentityForm = () => {
             placeholder={t.nationalIdInput.placeholder}
             label={`${t.nationalIdInput.label}:`}
           />
-          <TextInput
-            wrapperProps={createTestAttr(Ids.ssnInput)}
-            error={formState.errors.ssn?.message}
-            className={classes.textInput}
-            {...register('ssn')}
-            placeholder={t.ssnInput.placeholder}
-            label={`${t.ssnInput.label}:`}
+          <Select
+            wrapperProps={createTestAttr(Ids.genderInput)}
+            data={[
+              {
+                value: male.value,
```

Comment:
The value is a domain concern, we shouldn't keep it in our messages.

Reply:
So what about other select inputs?

Reply:
It's true for all select.
We should change it everywhere